### PR TITLE
changed layout

### LINKED ISSUE #9: Fix scroll on side panel
A scroll appears on the side panel if the viewport height is not enough to accommodate the entire content (profile, sections, and profile links). Handling for such a situation should be added.

### PR DESCRIPTION
**Pull request by A-Vashishth**

### CODE REVIEW

File: src/components/LeftPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import Profile from "./Profile";
+import SectionsList from "./SectionsList";
+
+const LeftPanel = () => {
+  return (
+    <Stack height="100%" id="left-panel-container">
+      <Box mb={10} id="profile-container" pt={8}>
+        <Profile />
+      </Box>
+      <Box id="list-container">
```
Comment (DecoderRony) [inline src/components/LeftPanel.tsx:11]: is it required to wrap sectionList in Box element?

Comment (A-Vashishth) [reply src/components/LeftPanel.tsx:11]: Not hard and fast required, but for consistency and readability, and as we discussed, I have kept this.

File: src/components/ColorModeSwitch.tsx [hunk 2]
```diff
@@ -26,13 +37,14 @@ const ColorModeSwitch = () => {
...
-    <Switch
+    <Icon
       zIndex="docked"
-      colorScheme="green"
-      isChecked={colorMode === "dark"}
-      onChange={handleColorModeChange}
-      size="lg"
-    />
+      onClick={handleColorModeChange}
+      fontSize={"2xl"}
```
Comment (HJ1X) [inline src/components/ColorModeSwitch.tsx:43]: We don't need curly braces here. Not a blocker but we can change this when possible.

**Review by HJ1X — APPROVED** [review]

**Comment by HJ1X** [comment]
Everything looked fine. I have observed a common pattern of using a JS template in props where just static values were passed. I've added a comment in the code for better to make it more clear.
Not a blocker though. We can fix that in the following commits.

**Review by DecoderRony — APPROVED** [review]

**Pull request merged**